internal/namespace/configreload.go

```diff
@@ -106,13 +106,6 @@ func (nm *namespaceManager) configReloaded(ctx context.Context) {
 		return
 	}
 
-	// Only initialize the updated namespaces (which includes all that depend on above plugins)
-	if err = nm.initNamespaces(ctx, updatedNamespaces); err != nil {
-		log.L(ctx).Errorf("Failed to initialize namespaces after config reload: %s", err)
-		nm.cancelCtx() // stop the world
-		return
-	}
-
```

Comment:
Trying to work out where the newly added namespaces go through the new retry loop? Sorry if I'm just not seeing it, but wanted to make sure we didn't miss a code path here

Reply:
Ah. yep I see it now. Sorry I just missed that the first time reading through.